### PR TITLE
ci: Adds workflow to publish docker builds

### PR DESCRIPTION
**Pull request by ryannikolaidis**

### CODE REVIEW

File: .terraform/environment [hunk 1]
```diff
@@ -0,0 +1 @@
+dev
```
Comment (cragwolfe) [inline .terraform/environment:1]: accidental file?

Comment (ryannikolaidis) [reply .terraform/environment:1]: yep, thanks

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "python.testing.pytestArgs": [
+        "test_unstructured"
+    ],
+    "python.testing.unittestEnabled": false,
+    "python.testing.pytestEnabled": true,
+    "githubPullRequests.ignoredPullRequestBranches": [
+        "main"
+    ]
+}
```
Comment (cragwolfe) [inline .vscode/settings.json:10]: this could be proposed in a separate PR, perhaps?

File: Dockerfile [hunk 3]
```diff
@@ -61,4 +69,4 @@ RUN python3.8 -m pip install pip==${PIP_VERSION} \
 
 COPY unstructured unstructured
 
-CMD ["/bin/bash"]
+CMD ["/bin/bash"]
```
Comment (cragwolfe) [inline Dockerfile:72]: newline nit per the github diff

File: .vscode/settings.json (hunk 2 above)
Comment (ryannikolaidis) [reply .vscode/settings.json:10]: ohh no, also, accidental

File: Dockerfile (hunk 3 above)
Comment (ryannikolaidis) [reply Dockerfile]: thanks!

**Review by cragwolfe — APPROVED** [review]
LGTM (even though PR is still in Draft mode : )

**Comment by ryannikolaidis** [comment]
> LGTM (even though PR is still in Draft mode : )

Thanks! Yea, was waiting for all the checkmarks and one final pass, much appreciated!
